tensorly/contrib/decomposition/tests/test_tt_TTOI.py

```diff
@@ -0,0 +1,48 @@
+
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Testing of applying TTOI
+
+@author: Lili Zheng
```

Comment:
Let's be consistent with the main file - we can put authorship in comment and no need for doctoring at this level for tests

Reply:
Sounds good! Authorship removed.